src/foam/nanos/crunch/MinMaxCapability.js

```diff
@@ -0,0 +1,80 @@
+foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'MinMaxCapability',
+  extends: 'foam.nanos.crunch.Capability',
+
+  javaImports: [
+    'foam.dao.ArraySink',
+    'foam.dao.DAO',
+    'java.util.List',
+    'static foam.mlang.MLang.*'
+  ],
+
+  properties: [
+    {
+      name: 'min',
+      class: 'Int',
+      value: 1,
```

Comment:
No trailing commas here and elsewhere.

Reply:
oops, not trying to break the styleguide it's just really hard to break the habit